tests/scripts/thread-cert/command.py

```diff
@@ -31,6 +31,18 @@ def check_address_notification(command_msg, source_node, destination_node):
     destination_rloc = destination_node.get_ip6_address(config.ADDRESS_TYPE.RLOC)
     assert ipv6.ip_address(destination_rloc) == command_msg.ipv6_packet.ipv6_header.destination_address, "Error: The IPv6 destination address is not the RLOC of the destination."
 
+def check_address_error_notification(command_msg, source_node, destination_address):
+    """Verify source_node sent a properly formatted Address Error Notification command message to destination_address.
+    """
+    command_msg.assertCoapMessageRequestUriPath('/a/ae')
+    command_msg.assertCoapMessageContainsTlv(network_layer.TargetEid)
+    command_msg.assertCoapMessageContainsTlv(network_layer.MlEid)
+
+    source_rloc = source_node.get_ip6_address(config.ADDRESS_TYPE.RLOC)
+    assert ipv6.ip_address(source_rloc) == command_msg.ipv6_packet.ipv6_header.source_address, "Error: The IPv6 source address is not the RLOC of the originator."
+
+    assert ipv6.ip_address(destination_address.decode('utf-8')) == command_msg.ipv6_packet.ipv6_header.destination_address, "Error: The IPv6 destination address is not expected."
+
```

Comment:
For above two `assert`, output more relative information for easier debug when error happens.